.travis.yml

```diff
@@ -13,11 +13,14 @@ matrix:
   include:
     - php: 7.1
       env: PHPSTAN=1
+           CODING_STANDARD=1
 
 before_script:
   - composer --prefer-source install
   - if [[ $PHPSTAN = 1 ]]; then composer require --dev phpstan/phpstan:^0.7; fi
+  - if [[ $CODING_STANDARD = 1 ]]; then composer require --dev symplify/easy-coding-standard:^2.2; fi
 
 script:
   - ./vendor/bin/phpunit
   - if [[ $PHPSTAN = 1 ]]; then vendor/bin/phpstan analyse -c phpstan.neon -l 3 lib tests; fi
+  - if [[ $CODING_STANDARD = 1 ]]; then vendor/bin/ecs check lib; fi
```

Comment:
Could you also apply the build stages feature that Travis now offers? It makes things so much better (in terms of configuration and execution). You can use the ORM as an example: https://github.com/doctrine/doctrine2/blob/a0c0d3bf2a7f49020ad7e54d7d3319a251574230/.travis.yml#L26